lib/rubocop/cop/style/guard_clause.rb

```diff
@@ -226,9 +226,14 @@ def range_of_branch_to_remove(node, guard)
           branch.source_range
         end
 
-        def guard_clause_source(guard_clause)
+        def guard_clause_exiting_keyword(guard_clause)
```

Comment:
I don't see the point in renaming this. Frankly, I find the new name confusing, so if you still want to change it should be something that reflects better the purpose of the method.